lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3998,7 +3998,7 @@ namespace Js
             OUTPUT_VERBOSE_TRACE(Js::DebuggerPhase, _u("DebugProfileProbeThunk: calling function: %s isWrapperRegistered=%d useDebugWrapper=%d\n"),
                 function->GetFunctionInfo()->HasBody() ? function->GetFunctionBody()->GetDisplayName() : _u("built-in/library"), AutoRegisterIgnoreExceptionWrapper::IsRegistered(scriptContext->GetThreadContext()), useDebugWrapper);
 
-            if (scriptContext->IsScriptContextInDebugMode())
+            if (scriptContext->IsScriptContextInDebugRecordingMode())
```

Comment:
Changed to IsDebuggerRecording. Need a method because of type visibility on DebugContext.

---
In reply to: [129986937](https://github.com/Microsoft/ChakraCore/pull/3447#discussion_r129986937) [](ancestors = 129986937)